docs/how-tos/performance-metrics.md

```diff
@@ -0,0 +1,134 @@
+You must run `viam-server` with `sudo` to monitor machine performance metrics.
+
+On macOS, you must also install telegraf by running `brew install telegraf` in your terminal before using this module.
+{{< /alert >}}
+
+## Add the telegraf-sensor
+
+{{< table >}}
+{{% tablestep link="/registry/configure/#add-a-modular-resource-from-the-viam-registry" %}}
+**1. Add the performance metrics sensor**
+
+On your machine's **CONFIGURE** page, click the **+** icon next to your machine part in the left-hand menu and select **Component**.
+
+Search for and add the `viam:viam-sensor:telegrafsensor` model provided by the [`viam-telegraf-sensor` module](https://app.viam.com/module/viam/viam-telegraf-sensor).
+
+Save the configuration.
+
+{{% /tablestep %}}
+{{% tablestep  %}}
+**2. (Optional) Customize the sensor configuration**
+
+To enable or disable specific metrics, add them to the attributes configuration.
+You can find a list of configurable attributes in the [module README](https://github.com/viamrobotics/viam-telegraf-sensor).
+For example:
+
+```json
+{
+  "disable_kernel": true
+}
+```
+
+Save the configuration.
+
+{{% /tablestep %}}
+{{% tablestep  %}}
+**3. Test the sensor**
+
+After you configure your sensor, click **Test** at the bottom of the sensor configuration card to view the readings.
+You can also see readings on the **CONTROL** tab.
+
+![Test panel with readings displayed.](/how-tos/telegraf-test.png)
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Configure the data management service
+
+To capture the data from your configured sensor, you need to add the [data management service](/services/data/) and configure it to capture and sync the sensor data:
+
+{{< table >}}
+{{% tablestep link="/services/data/" %}}
+**1. Add the data management service**
+
+On your machine's **CONFIGURE** page, click the **+** icon next to your machine part in the left-hand menu and select **Service**.
+
+Select the `data management / RDK` service and click **Create**.
+You can leave the default data sync interval of `0.1` minutes to sync every 6 seconds.
+Also leave both **Capturing** and **Syncing** toggles in the "on" position.
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**2. Configure data capture on the telegraf sensor**
+
+Return to your `telegrafsensor`'s configuration card.
+
+In the **Data capture** section, click **Add method**.
+
+From the **Method** dropdown select `Readings`.
+Set the **Frequency** to `0.05` Hz to capture readings once every 20 seconds.
+
+![Sensor readings capture configuration.](/how-tos/capture-readings.png)
+
+Save your config.
+{{% /tablestep %}}
+{{< /table >}}
+
+## View synced data
+
+Click the **...** menu in the upper-right corner of the sensor configuration card.
+Select **View captured data**.
+If you do not immediately see data, wait a minute for the data to be captured and synced at the intervals you specified, then refresh the page.
+
+![View of sensor data](/services/data/sensor-data.png)
+
+### Next steps
```

Comment:
```suggestion
## Stop data capture

If this is a test project, make sure you stop data capture to avoid charges for a large amount of unwanted data.

In the **Data capture** section of your sensor's configuration, toggle the switch to **Off**.

Click the **Save** button in the top right corner of the page to save your config.

### Next steps
```

Reply:
Okay adding. Also now I noticed next steps should be h2.
```suggestion
## Stop data capture

If this is a test project, make sure you stop data capture to avoid charges for a large amount of unwanted data.

In the **Data capture** section of your sensor's configuration, toggle the switch to **Off**.

Click the **Save** button in the top right corner of the page to save your config.

## Next steps
```